core/metrics-core-impl/src/main/java/org/hawkular/metrics/core/impl/MetricsServiceImpl.java

```diff
@@ -227,6 +231,30 @@ void unloadDataRetentions() {
         dataRetentions.clear();
     }
 
+    private void initSystemTenant() {
+        CountDownLatch latch = new CountDownLatch(1);
+        Trigger trigger = new RepeatingTrigger.Builder().withInterval(30, MINUTES).withDelay(30, MINUTES).build();
+        dataAccess.insertTenant(new Tenant(SYSTEM_TENANT_ID))
+                .filter(ResultSet::wasApplied)
+                .map(row -> taskScheduler.scheduleTask(CreateTenants.TASK_NAME, SYSTEM_TENANT_ID, 100, emptyMap(),
+                        trigger))
+                .subscribe(
+                        task -> logger.debug("Scheduled {}", task),
+                        t -> {
+                            logger.error("Failed to initialize system tenant", t);
```

Comment:
Yeah, this should probably cause start up to fail. ```initiSystemTenant``` should probably return an Observable rather than void. I will create a ticket to address this.